lib/fathead/node_js/parse.js

```diff
@@ -33,20 +33,19 @@ function getAbstract(obj) {
     var a = obj.desc ? '<p>' + obj.desc + '</p>' : '',
         code = '',
         stability = '';
+    // strip <a> and <p> HTML tags but keep others.
+    a = a.replace(/<\s*p*>/ig, '');
```

Comment:
However, what we don't want is any empty `<p></p>` in the abstract. We shouldn't be creating any of those 👍